src/sonic-yang-mgmt/sonic_yang.py

```diff
@@ -0,0 +1,450 @@
+import yang as ly
+import sys
+import json
+import glob
+
+"""
+Yang schema and data tree python APIs based on libyang python
+"""
+class sonic_yang:
+    def __init__(self, yang_dir):
+        self.yang_dir = yang_dir
+        self.ctx = None
+        self.module = None
+        self.root = None
+
+        try:
+            self.ctx = ly.Context(yang_dir)
+        except Exception as e:
+            print(e)
```

Comment:
We can add little more details here, [min class name]

Reply:
Error messages added